src/components/actions/processTrigger.ts

```diff
@@ -0,0 +1,101 @@
+import { ActionInTemplate, TriggerPayload, ActionSequential } from '../../types'
+import DBConnect from '../databaseConnect'
+import { actionLibrary } from '../pluginsConnect'
+import { EvaluatorNode } from '@openmsupply/expression-evaluator/lib/types'
+import { executeAction } from './executeAction'
+import { ActionQueueStatus, TriggerQueueStatus } from '../../generated/graphql'
+import { swapOutAliasedActions } from './helpers'
+
+// Dev config
+const showActionOutcomeLog = false
```

Comment:
process.env ?

Reply:
as in maybe we should have like a `debug` env variable to toggle this ?

Reply:
Maybe, but you'd want to have some fine-grained control over what gets printed. Wouldn't want to enable `debug` and have a whole lot of big objects scroll past when you're only interested in one of them.

But yes, in general, sounds like a good idea.